Packs/PrismaSaasSecurity/Integrations/SaasSecurity/SaasSecurity.py

```diff
@@ -0,0 +1,493 @@
+                                            params=params, headers=headers)
+        return token_response.get('access_token')
+
+    def get_incidents(self, limit: int = None, from_time: str = None, to_time: str = None, app_ids: str = None,
+                      state: str = 'open', severity: str = None, status: str = None, next_page: str = None):
+        """
+        :param limit: The number of incidents to pull per page. Default is 50, max is 1000, min is 10.
+        :param from_time: The start time of query, filter by the incident's “updated-at” field.
+        :param to_time: The end time of query, filter by the incident's “updated-at” field.
+        :param app_ids: List of application id. Comma-separated.
+        :param state: The state of the incidents to pull. Default is open.
+        :param severity: The severity of the incidents to pull.
+        :param status: The status of the incidents to pull.
+        :param next_page: For pagination purposes. If provided, params should be None.
+        """
+        url_suffix = next_page or '/incident/api/incidents/delta'
+        state = state if state != 'All' else None
+
+        params = {
+            'limit': limit,
+            'from': from_time,
+            'to': to_time,
+            'app_ids': app_ids,
+            'state': state,
+            'severity': severity,
+            'status': status,
+        } if not next_page else {}
+        remove_nulls_from_dictionary(params)
+
+        return self.http_request('GET', url_suffix=url_suffix, params=params)
+
+    def get_incident_by_id(self, inc_id: str):
+        """
+        :param inc_id: The incident ID.
+        """
+        return self.http_request('GET', url_suffix=f'/incident/api/incidents/{inc_id}')
+
+    def update_incident_state(self, inc_id: str, category: str):
+        """
+        :param inc_id: The incident ID.
+        :param category: Closing category.
+        """
+        body = {
+            'state': 'closed',
+            'category': category,
+        }
+        return self.http_request('POST', url_suffix=f'/incident/api/incidents/{inc_id}/state', json_data=body)
+
+    def get_apps(self):
+        return self.http_request('GET', url_suffix='/incident/api/apps')
+
+    def remediate_asset(self, asset_id: str, remediation_type: str, remove_inherited_sharing: bool):
+        """
+        :param asset_id: The asset ID.
+        :param remediation_type: The remediation action to take.
+        :param remove_inherited_sharing: Used when remediation type is “remove_public_sharing”,
+            when set to true, all the parent folder sharing url will be removed.
+        """
+        body = assign_params(
+            remediation_type=remediation_type,
+            remove_inherited_sharing=remove_inherited_sharing,
+            asset_id=asset_id,
+        )
+
+        self.http_request('POST', url_suffix='/remediation/api/assets', json_data=body, resp_type='response')
```

Comment:
why do you need `resp_type` as a response?

Reply:
there's not response for this case (not even a text)